src/prefabs/pageWithProfile.tsx

```diff
@@ -458,6 +459,75 @@ const beforeCreate = ({
     return undefined;
   }
 
+  const profilePictureDisabledKinds = createBlacklist([
+    'BELONGS_TO',
+    'GOOGLE_DOCUMENT',
+    'HAS_ONE',
+    'IMAGE',
+    'LOGIN_TOKEN',
+    'OBJECT',
+  ]);
+
+  const profileNameDisabledKinds = createBlacklist([
```

Comment:
Some of these whitelisted props make no sense: Login token, google document, etc. Also remove object property here